homeworks/olena.ambitska_elena-ambitska/GamesProject/success.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" href="styles/main.css">
+        <title>Success</title>
+    </head>
+    <body>
+        <img src="./images/unicorn.gif" alt="success unicorn image" class="success__picture">
```

Comment:
Hey, @elena-ambitska, it's so sweeeet🥰
But please add button `Return to home page` according to the [task description](https://github.com/MastersAcademy/fe-course-basic-2023/issues/137#issue-1955947368), point 3.

Reply:
Hey, @vladyslav-yermolin-moc :raised_hands:
Added go-back button. Something went wrong while I was reading the task(